addons/mail/models/mail_activity.py

```diff
@@ -49,6 +49,18 @@ def default_get(self, fields):
         ('default', 'Other')], default='default',
         string='Category',
         help='Categories may trigger specific behavior like opening calendar view')
+    mail_template_ids = fields.Many2many('mail.template', string='Mails templates')
+    initial_res_model_id = fields.Many2one('ir.model', 'Initial model', compute="_get_initial_res_model_id", store=False, help='Technical field tokeep trace of the model at the beginning of the edition'
```

Comment:
I would put a small comment to separate real fields from display fields so that we know quickly those fields are not really used for the model itself.